starlette/staticfiles.py

```diff
@@ -146,17 +146,18 @@ async def get_response(self, path: str, scope: Scope) -> Response:
                 return self.file_response(full_path, stat_result, scope)
 
         if self.html:
```

Comment:
Is it too crazy to change the type of `html` to `bool | Literal[200, 404]` ?

Or to include a `html_status_code` parameter?